sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisErrorSuite.scala

```diff
@@ -117,8 +117,18 @@ class AnalysisErrorSuite extends AnalysisTest {
       plan: LogicalPlan,
       errorClass: String,
       messageParameters: Array[String]): Unit = {
+    errorClassTest(name, plan, errorClass, null, messageParameters)
+  }
+
+  def errorClassTest(
+    name: String,
+    plan: LogicalPlan,
+    errorClass: String,
+    errorSubClass: String,
+    messageParameters: Array[String]): Unit = {
```

Comment:
wrong indentation, see https://github.com/databricks/scala-style-guide#spacing-and-indentation

Reply:
Done, thank you